IDE/Espressif/ESP-IDF/libs/CMakeLists.txt

```diff
@@ -30,7 +30,6 @@ endif()
 set(COMPONENT_SRCEXCLUDE
     "wolfcrypt/src/aes_asm.S"
     "wolfcrypt/src/evp.c"
-    "wolfcrypt/src/misc.c"
     "src/bio.c"
     )
```

Comment:
I think that your settings would have NO_INLINE definition. Current user_settings.h for ESP-IDF port doesn't have NO_INLINE definition as default.  The misc.c is included when expanding to INLINE. Therefore, the file excludes from target file in makefile. 
Because the user_settings.h is good for default, I think that we don't need to remove this.